### PR TITLE
add extract/rollback tools

### PR DESCRIPTION
**Pull request by FletcherMan**
## Description

Closes: #XXX

______

**All** items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.

PR review checkboxes:

I have...

- [ ] added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/teleport-network/teleport/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link in the PR description to the relevant issue or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all required CI checks have passed

Code maintenance:

I have...

- [ ] written unit and integration [tests](https://github.com/teleport-network/teleport/blob/main/CONTRIBUTING.md#testing)
- [ ] added relevant [`godoc`](https://go.dev/blog/godoc) and [code comments](https://blog.jbowen.dev/2019/09/the-magic-of-go-comments/).
- [ ] updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)

______

### Reviewers Checklist

**All** items are required. Please add a note if the item is not applicable and please add your handle next to the items reviewed if you only reviewed selected items.

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed all author checklist items have been addressed
- [ ] confirmed that this PR does not change production code

### CODE REVIEW

File: tools/cmd.go [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+			if len(targetHome) == 0 {
+				return errors.New("missing target-home")
+			}
+			cfg := server.GetServerContextFromCmd(cmd).Config
+			home := cfg.RootDir
+			extract.CopyBlockStore(home, targetHome)
+			return nil
+		},
+	}
+	cmd.Flags().String("target-home", "", "The home path of the target db ")
```
Comment (KaelTop) [inline tools/cmd.go:53]: ditto.

**Review by bradyjoestar — APPROVED** [review]

File: tools/cmd.go [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+			targetHome, _ := cmd.Flags().GetString("target-home")
+			if len(targetHome) == 0 {
+				return errors.New("missing target-home")
+			}
+			cfg := server.GetServerContextFromCmd(cmd).Config
+			home := cfg.RootDir
+			return extract.CopyApplication(home, targetHome, storeKeys)
+		},
+	}
+	cmd.Flags().String("target-home", "", "The home path of the target db ")
```
[the remark this answers is not in the document]
Comment (FletcherMan) [reply tools/cmd.go]: fixed

[the remark this answers is not in the document]
Comment (FletcherMan) [reply tools/cmd.go]: fixed

File: tools/cmd.go (hunk 1 above)
Comment (FletcherMan) [reply tools/cmd.go]: fixed

**Review by Zer0ne001 — APPROVED** [review]

**Pull request merged**